paasta_tools/setup_prometheus_adapter_config.py

```diff
@@ -288,6 +304,96 @@ def create_instance_uwsgi_scaling_rule(
     }
 
 
+def create_instance_piscina_scaling_rule(
+    service: str,
+    instance: str,
+    autoscaling_config: AutoscalingParamsDict,
+    paasta_cluster: str,
+) -> PrometheusAdapterRule:
+    """
+    Creates a Prometheus adapter rule config for a given service instance.
+    """
+    setpoint = autoscaling_config["setpoint"]
+    moving_average_window = autoscaling_config.get(
+        "moving_average_window_seconds", DEFAULT_UWSGI_AUTOSCALING_MOVING_AVERAGE_WINDOW
+    )
+    # this should always be set, but we default to 0 for safety as the worst thing that would happen
+    # is that we take a couple more iterations than required to hit the desired setpoint
+    offset = autoscaling_config.get("offset", 0)
```

Comment:
do we end up using a piscina worker/task queue slot/whatever the correct terminology is here to serve the metrics that prometheus scrapes? if not, we can get rid of `offset` (this is meant to correct for a uwsgi worker being utilized to serve metrics (e.g., if you have 4 uwsgi workers - ~1 worker will always be shown as busy even when you're 100% idle since hitting the metrics endpoint requires a worker to serve that data))

Reply:
Oh no, we are just using the main process in Fastify to serve the metrics. I'll remove it.